data/data/bootstrap/baremetal/files/usr/local/bin/master-bmh-update.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
+
+# shellcheck disable=SC1091
+. /usr/local/bin/release-image.sh
+
+export KUBECONFIG=/etc/kubernetes/kubeconfig
+
+# Wait till the baremetalhosts are populated
+until oc get baremetalhosts -n openshift-machine-api; do
+   echo Waiting for BareMetalHosts to appear...
+   sleep 20
+done
+
+# Wait for a master to appear.
+while [ "$(curl -s http://localhost:6385/v1/nodes | jq '.nodes[] | .uuid' | wc -l)" -lt 1 ]; do
```

Comment:
also do we expect the RHCOS image to ship jq ? that's odd.

Reply:
Why’s that odd? It’s there.